quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/ShowConfig.java

```diff
@@ -154,10 +133,6 @@ private void printProperty(String property) {
     }
 
     private static String groupProperties(String property) {
-        if (property.startsWith("%")) {
```

Comment:
@shawkins Just out of curiosity - What was the purpose of using '%'? And why it's gonna be removed now? It should mark a profile for the config property, right?

Reply:
In this instance % properties aren't emited in any case by the surrounding logic - only those properly prefixed by quarkus. or keycloak.

More generally there are a couple of places where we handling profile based property names - however the current behavior of smallrye makes me think that is highly dependent on how you are accessing the configuration. From a top-level you never see those names - they will have already been resolved against the profile.